ansible/files/bootstrap-flatcar.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 source /etc/os-release
 
-[[ "${NAME,,}" != *"flatcar"* ]] && exit 0
+[[ "${ID,,}" != *"flatcar"* ]] && exit 0
```

Comment:
Maybe we can be a little safer here? (I don't know what the actual syntax would be)

```suggestion
[[ ["${ID,,}" != *"flatcar"*] && ["${NAME,,}" != *"flatcar"*]  ]] && exit 0
```

Reply:
these are current contents of /etc/os-release
```
NAME="Flatcar Container Linux by Kinvolk"
ID=flatcar
ID_LIKE=coreos
VERSION=3033.3.16
VERSION_ID=3033.3.16
BUILD_ID=2023-08-07-1641
PRETTY_NAME="Flatcar Container Linux by Kinvolk 3033.3.16 (LTS 2022)"
ANSI_COLOR="38;5;75"
HOME_URL="https://flatcar.org/"
BUG_REPORT_URL="https://issues.flatcar.org"
FLATCAR_BOARD="amd64-usr"

```

Reply:
existing script works as expected. reverted this code change.